packages/gatsby/cache-dir/ssr-develop-static-entry.js

```diff
@@ -237,7 +241,7 @@ export default async function staticPage(
           !isClientOnlyPage
         ) {
           pageElement = React.createElement(
-            syncRequires.ssrComponents[componentChunkName],
+            preferDefault(syncRequires.ssrComponents[componentChunkName]),
```

Comment:
`preferDefault` removed from `ssr-sync-requires` (to allow bundling and executing user's `getServerData`) so we use it here